generators/utilities/src/main/resources/csrc/emulator.cc

```diff
@@ -235,6 +246,7 @@ int main(int argc, char** argv)
             c = '?';
           } else {
             c = 'p';
```

Comment:
This should be a capital `P`.
We have found a verilog plusarg not the argument `+permissive` which is what lowercase `p` means.

Reply:
For my sanity I tried to derive a case where this would manifest but,
This bug appears to be undetectable because you only arrive to this case if `opterr` is already 0 so the side effect of the lowercase `p` is already in effect.
However the intent is clearly that we should be setting this to capital `P`

Reply:
Fixed.